runtimes/rust/lbr-prelude/Cargo.toml

```diff
@@ -2,8 +2,10 @@
 name = "lbr-prelude"
 version = "0.1.0"
 edition = "2021"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+license = "Apache-2.0"
+homepage = "https://mlabs-haskell.github.io/lambda-buffers/"
+description = "Basic data types and utilities for lambda-buffers"
```

Comment:
nit: LambdaBuffers runtime library for the Prelude schema.

Reply:
```suggestion
description = "LambdaBuffers runtime library for the Prelude schema."
```